llvm/test/Transforms/LICM/hoist-binop.ll

```diff
@@ -409,59 +405,56 @@ entry:
 loop:
   %index = phi i64 [ 0, %entry ], [ %index.next, %loop ]
   %step.add = add nuw nsw i64 %index, %c1
-  call void @use(i64 %step.add)
   %index.next = add nsw i64 %step.add, %c2
   br label %loop
 }
 
-define void @add_both_nsw_second_nuw(i64 %c1, i64 %c2) {
-; CHECK-LABEL: @add_both_nsw_second_nuw(
+;
+; Hoist MUL and drop NSW even if both ops have it.
+define void @mul_no_nsw_2(<2 x i64> %c1, <2 x i64> %c2) {
```

Comment:
Could you move this below `@add_both_nsw_second_nuw` (where I believe it used to be before) to simplify the diff?